### ISSUE TITLE
prettify converts decimals to int

### ISSUE THREAD

**Issue by skeletorkun** [chrismeyers/vscode-pretty-json#2]
before : 

`{"driving":8.0,"stopover":8.0,"trip":8.0}`

after

```
{
    "driving": 8,
    "stopover": 8,
    "trip": 8
}
```

**Comment by chrismeyers** [comment]
It looks like this is just the nature of how `JSON.stringify` / `JSON.parse` work.  Using non-zero floating point values such as `{"driving":8.1,"stopover":8.2,"trip":8.3}` result in the expected output of:

```json
{
  "driving": 8.1,
  "stopover": 8.2,
  "trip": 8.3
}
```

The only other thing I can think of is storing the data as a string, like `{"driving":"8.0","stopover":"8.0","trip":"8.0"}`.

**Comment by chrismeyers** [comment]
I just published v2.3.0 which addresses this issue (see https://github.com/chrismeyers/vscode-pretty-json/commit/9872209245e5c56d2e8f4030e709cf2ae8bddc61)
